src/App.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <pokemonCards
```

Comment:
Use pokemonCards for showing pokemonEvolve @PiyushInnotech

Reply:
means in the app.vue there will be 2 pokemonCards, 1 for showing pokemon another for evolitions @PiyushInnotech